src/lib/orionTypes/EntityTypesResponse.cpp

```diff
@@ -66,11 +66,6 @@ std::string EntityTypesResponse::render(ConnectionInfo* ciP, const std::string&
   std::string out                 = "";
   std::string tag                 = "entityTypesResponse";
 
-  if ((ciP->uriParam["attributesFormat"] == "object") && (ciP->outFormat == JSON))
```

Comment:
Currently, CNR includes:

"Fix: simplifying 'GET /v1/contextTypes' and 'GET /v1/contextTypes/<entityType>' operations, removing attribute types (no longer used as attribute identification since 0.17.0)"

Do you think is enought or should we explictely mention the attributeFormat thing?